src/Plugin/Field/FieldFormatter/FieldFormatterBase.php

```diff
@@ -15,7 +15,7 @@
   /**
    * Entity view display.
    *
-   * @var \Drupal\Core\Entity\Display\EntityViewDisplayInterface
+   * @var \Drupal\Core\Entity\Display\EntityViewDisplayInterface[]
```

Comment:
Hi Andypost!

Are you sure about this point?
 https://github.com/drupal-media/field_formatter/blob/8.x-1.x/src/Plugin/Field/FieldFormatter/FieldFormatterFromViewDisplay.php#L119 
https://github.com/drupal-media/field_formatter/blob/8.x-1.x/src/Plugin/Field/FieldFormatter/FieldWrapperBase.php#L260